python/pyspark/sql/dataframe.py

```diff
@@ -519,7 +520,7 @@ def sort(self, *cols, **kwargs):
             jcols = [jc if asc else jc.desc()
                      for asc, jc in zip(ascending, jcols)]
         else:
-            raise TypeError("ascending can only be bool or list, but got %s" % type(ascending))
+            raise TypeError("ascending can only be boolean or list, but got %s" % type(ascending))
```

Comment:
It's weird to see we put `boolean` and `int` together, it should be either `boolean/integer` or `bool/int`.